pcdsdevices/beam_stats.py

```diff
@@ -26,3 +28,63 @@ class SxrGmd(Device):
 
     def __init__(self, prefix='', name='sxr_gmd', **kwargs):
         super().__init__(prefix=prefix, name=name, **kwargs)
+
+
+class BeamEnergyRequest(PVPositionerIsClose):
+    """
+    Positioner to request beam color changes from ACR in eV.
+
+    It is up to ACR how to and whether to fulfil these requests. This is often
```

Comment:
```suggestion
    It is up to ACR how to and whether to fulfill these requests. This is often
```

Reply:
TIL fulfil/fulfill has two spellings https://www.grammarly.com/blog/fulfil-fulfill/

Reply:
We will go with the US spelling as suggested

Reply:
2nded that TIL... huh. It jumped out with the `fulfilled` below